### PR TITLE
comunity_topics_workflow: add a reminder about milestones

### PR DESCRIPTION
**Pull request by Andersson007**
Relates to https://github.com/ansible-community/community-topics/issues/122#issuecomment-1197132072

### CODE REVIEW

**Review by mariolenz — APPROVED** [review]
LGTM

**Review by mariolenz — CHANGES_REQUESTED** [review]
Maybe we should clarify the status of the topic when it implies actions related to a future release: When an issue is added to the corresponding milestone, is the topic `implemented` or still `being_implemented` until it's really implemented in the future release?

Personally, I tend to consider it `implemented` in that case. Or, alternatively, have another label to say that this topic is more or less resolved, even if not really implemented yet. Like `scheduled_for_future_release` or similar. And then close it and move it to the `Resolved` column. After all, there's nothing much to do with the _topic_, the doing is tracked in the milestone.

Whether you agree with me or not, I think we should document here what should happen to the topic.

**Comment by Andersson007** [comment]
> Maybe we should clarify the status of the topic when it implies actions related to a future release: When an issue is added to the corresponding milestone, is the topic `implemented` or still `being_implemented` until it's really implemented in the future release?
> 
> Personally, I tend to consider it `implemented` in that case. Or, alternatively, have another label to say that this topic is more or less resolved, even if not really implemented yet. Like `scheduled_for_future_release` or similar. And then close it and move it to the `Resolved` column. After all, there's nothing much to do with the _topic_, the doing is tracked in the milestone.
> 
> Whether you agree with me or not, I think we should document here what should happen to the topic.

Good catch! I'm fine with `implemented` + `scheduled_for_future_release`. @felixfontein what do you think? (i don't think we should vote on this or discuss it broadly)

**Comment by felixfontein** [comment]
Sounds good to me!

**Comment by Andersson007** [comment]
@mariolenz @felixfontein added a step about labeling, PTAL

File: community_topics_workflow.md [hunk 1]
```diff
@@ -60,4 +60,9 @@ After a topic is created, a Steering Committee member (hereinafter `Committee pe
   - [ ] Comments on the topic that the work is done.
   - [ ] Removes the `being_implemented` label.
   - [ ] Add the `implemented` label.
+- [ ] If the topic implies actions related to the future Ansible Community package releases (for example, a collection exclusion), the Committee person:
+
+  - [ ] Adds the `scheduled_for_future_release` label to the topic.
+  - [ ] Checks if there's a corresponding milestone in the [ansible-build-data](https://github.com/ansible-community/ansible-build-data/milestones) repository. If there's no milestone, the person creates it.
+  - [ ] Creates an issue and adds it to the milestone.
```
Comment (felixfontein) [inline community_topics_workflow.md:67]:
```suggestion
  - [ ] Creates an issue in ansible-build-data that references the topic in community-topics, and adds it to the milestone.
```

Comment (Andersson007) [reply community_topics_workflow.md]: SGTM, committed, thanks, PTAL

**Review by mariolenz — APPROVED** [review]
LGTM

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@mariolenz @felixfontein thanks for reviewing!

**Pull request merged**